### PR TITLE
Metric sidebar

### PR DESCRIPTION
**Pull request by OAGr**
image

### CODE REVIEW

File: src/components/metrics/card/token/index.js [hunk 1]
```diff
@@ -10,13 +10,13 @@ const MetricReadableId = ({readableId}) => (
   </div>
 )
 
-const MetricExpandButton = ({onOpenModal}) => (
+const MetricExpandButton = ({onOpenSidebar}) => (
```
Comment (mmcdermott) [inline src/components/metrics/card/token/index.js]: rename

File: src/components/metrics/card/index.js [hunk 2]
```diff
@@ -214,6 +224,13 @@ export default class MetricCard extends Component {
     return !!(isAnalysis && selectedMetric && this._shouldShowSimulation(metric) && this._shouldShowSimulation(selectedMetric))
   }
 
+  onMouseDown(e){
+    const isSidebarElement = (_.get(e, 'target.dataset.controlSidebar') === "true")
+    if (this.state.sidebarIsOpen && !isSidebarElement){
```
Comment (mmcdermott) [inline src/components/metrics/card/index.js]: What does this do? And why not use toggleSidebar here.

File: src/components/metrics/card/token/index.js (hunk 1 above)
Comment (mmcdermott) [inline src/components/metrics/card/token/index.js]: I meant rename `MetricExpandButton` to something related to the sidebar menu.

**Comment by OAGr** [comment]
This completes the issue: https://github.com/getguesstimate/guesstimate-app/issues/746

**Pull request merged**